### PR TITLE
add ok prop for response

### PR DESCRIPTION
**Pull request by bobenut**
Hi
react-fetch-mock is very useful for me. thank u.
Sometimes it is necessary to check if response is ok using Response.ok.
According to fetch api specification. I added ‘ok’ prop into response object.
fetch specification:
https://developer.mozilla.org/en-US/docs/Web/API/Response/ok

### CODE REVIEW

**Comment by bobenut** [comment]
but i haven't compile into dist.
what do you think about it/

File: src/index.js [hunk 1]
```diff
@@ -135,6 +135,7 @@ class FetchMock {
       obj = {
         status: 200,
         data: obj,
+        ok: true
```
Comment (WhatAKitty) [inline src/index.js:138]: It is unnecessary

**Comment by WhatAKitty** [comment]
@bobenut I'll do it after I merged this pr.

**Comment by bobenut** [comment]
ok thx

File: src/index.js (hunk 1 above)
Comment (bobenut) [reply src/index.js:138]: got it

**Comment by WhatAKitty** [comment]
@bobenut 
https://github.com/WhatAKitty/react-fetch-mock/commit/207cd6ebffdaf5baa12c39615ebaf48d46635cbb

**Pull request closed without merging**